src/gtk/settings.cpp

```diff
@@ -26,6 +26,19 @@
 #include "wx/gtk/private/stylecontext.h"
 #include "wx/gtk/private/value.h"
 
+#define PORTAL_BUS_NAME "org.freedesktop.portal.Desktop"
+#define PORTAL_OBJECT_PATH "/org/freedesktop/portal/desktop"
+#define PORTAL_SETTINGS_INTERFACE "org.freedesktop.portal.Settings"
+
+enum ColorScheme
```

Comment:
Interesting, the only documentation for this I could find was https://docs.flatpak.org/fr/latest/portal-api-reference.html
but this doesn't seem to be flatpak-specific. Do you know of some better/more authoritative documentation for this? I'd like to add a link to it to a comment here somewhere.

Reply:
The xdg-desktop-portal code is hosted on [github](https://github.com/flatpak/xdg-desktop-portal) which links to: https://flatpak.github.io/xdg-desktop-portal/